solr/solr-ref-guide/src/meta-docs/publish.adoc

```diff
@@ -47,54 +49,20 @@ To build the HTML:
 [source,bash]
 $ ant clean default
 +
-This will produce pages with a DRAFT watermark across them. While these are fine for initial DRAFT publication, see the section <<Publish the Guide>> for steps to produce final production-ready HTML pages.
+This will produce pages with a DRAFT watermark across them. While these are fine for initial DRAFT publication, see the section <<Publish the Guide without watermark>> for steps to produce final production-ready HTML pages.
 . The resulting Guide will be in `solr/build/solr-ref-guide`. The HTML files themselves will be in `solr/build/solr-ref-guide/html-site`.
 
 == Upload to the Website
 
-// A lot of this was copied from https://wiki.apache.org/lucene-java/ReleaseTodo#Website_.2B-.3D_javadocs. See that section for explanations for why some steps are required.
-
-*Step 1: Update extpaths.txt in CMS Staging*
-
-. Checkout CMS trunk:
-+
-[source,bash]
-$ svn co --depth=immediates https://svn.apache.org/repos/asf/lucene/cms/trunk/content website-source
-+
-* If you already have this repo checked out, you can simply `svn up website-source` to update to the latest revision.
-. `$ cd website-source` (where you just checked out the repo)
-. Add the path for the new Guide version:
-+
-[source,bash]
-$ echo solr/guide/X_Y >> extpaths.txt
-+
-where `X_Y` is the new version (i.e, `7_7`)
-. Commit changes:
-+
-[source,bash]
-$ svn commit -m "Update CMS production sync exceptions for X_Y_Z Guide" extpaths.txt
-
-*Step 2: Push Guide to Website Production*
-
 Push the Guide directly to production via Subversion `import` from where you built it.
 
-You must push it to the path you just added to `extpaths.txt`, so if the path you added was `solr/guide/7_7`, you'll use the path as shown in the below example:
-
 [source,bash]
 svn -m "Add Ref Guide for Solr 7.7" import <checkoutroot>/solr/build/solr-ref-guide/html-site https://svn.apache.org/repos/infra/websites/production/lucene/content/solr/guide/7_7
 
 Confirm you can browse to Guide manually by going to the new URL. For example:
 https://lucene.apache.org/solr/guide/7_7
 
-*Step 3: Push Staging extpaths.txt to Production*
-
-The `extpaths.txt` works by listing paths that should be ignored when the CMS syncs the staging and production repositories. Publishing staging to production will only succeed if the paths listed in `extpaths.txt` exist in production. At the same time, if a path exists in production but not in staging it will be deleted unless it is defined in `extpaths.txt`.
-
-After pushing the content to production, check that the `extpaths.txt` in production includes the proper path to ensure that the Guide is not deleted incorrectly. If it does not exist in production, try to publish the site again to make sure it is updated.
-
-Production URL: https://lucene.apache.org/extpaths.txt
-
-== Publish the Guide
+== Publish the Guide without watermark
```

Comment:
Headline case in section titles please....

But not sure why this needed to change? I don't care, just struck me as extraneous and then I realized that the other section heading doesn't say you're publishing a draft so it probably really doesn't matter. I'm the only one who reads this anyway.

Reply:
Reverted to old header